src/components/navigation/Navigation.js

```diff
@@ -0,0 +1,126 @@
+/**
+ *
+ * @author Sriram
+ * A component that displays app level navigation
+ */
+import React from "react";
+import intlData from "./messages";
+import ids from "./ids";
+import { build, getMessage, withI18N } from "@cyverse-de/ui-lib";
+import CloudUploadIcon from "@material-ui/icons/CloudUpload";
+import AppsIcon from "@material-ui/icons/Apps";
+import DashboardIcon from "@material-ui/icons/Dashboard";
+import AssessmentIcon from "@material-ui/icons/Assessment";
+import MoreHorizIcon from "@material-ui/icons/MoreHoriz";
+import { Button, Grid, Hidden, Paper } from "@material-ui/core";
+import { useRouter } from "next/router";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import { injectIntl } from "react-intl";
+
+const useStyles = makeStyles((theme) => ({
+    button: {
+        margin: 1,
+        "&:hover": {
+            backgroundColor: theme.palette.blue,
+            color: theme.palette.white,
+        },
+    },
+
+    selectedButton: {
+        margin: 1,
+        backgroundColor: theme.palette.blue,
+        color: theme.palette.white,
+    },
+}));
+
+const DASHBOARD = "dashboard";
+const DATA = "data";
+const APPS = "apps";
+const ANALYSES = "analyses";
+const MORE = "more";
+
+function Navigation(props) {
+    const classes = useStyles();
+    const router = useRouter();
+    const { navigate } = props;
+    const theme = useTheme();
+    return (
+        <Paper elevation={10} style={{ backgroundColor: theme.palette.lightGray }}>
+            <Grid container>
+                <Grid item xs>
+                    <Button
+                        id={build(ids.NAVIGATION, ids.DASHBOARD_BTN)}
+                        className={
+                            navigate === DASHBOARD
+                                ? classes.selectedButton
+                                : classes.button
+                        }
+                        onClick={() => {
+                            router.push("/dashboard");
```

Comment:
I'm not sure if this could end up being more trouble than it's worth, but with the data listing, I had the page control the routing instead of the component.  My thinking was if we need to change the routing for whatever reason (like maybe `analyses` needs to get changed to `jobs` because people don't know the difference between that and `apps`, just as an example), we don't have to go digging through the components.  Again, not sure if that's worth all the potential passing around or not.  Not something we have to decide now though.

Reply:
Not sure if I follow you here. There is no corresponding page for Navigation component 🤔 . My thinking was this Navigation component will handle all the logic for navigating from one view to the other.

Reply:
Oh, right 😊 Hm... although that makes me wonder, is there not a reason to put the navigation only in the `_app` page like the `CyVerseAppBar`?

Reply:
Thats possible. I had it that way at one point but changed it for some reason. I can give that a shot again.